bundles/mapping/layerswipe/instance.js

```diff
@@ -16,6 +16,7 @@ Oskari.clazz.define(
         this.cropSize = null;
         this.mapModule = null;
         this.layer = null; // ol layer
+        this.oskariLayer = null;
```

Comment:
Rename so it's instantly clear it holds the id and not the layer object:
```suggestion
        this.oskariLayerId = null;
```